modules/web-console/frontend/app/modules/agent/AgentManager.service.ts

```diff
@@ -475,7 +478,7 @@ export default class AgentManager {
 
                         const data = await this.pool.postMessage({payload: res.data, useBigIntJson});
 
-                        return data.result ? data.result : data;
+                        return event === EVENT_REST ? data : data.result;
```

Comment:
May be we should instead always send data from Web Agent as "data.result"?
@nva What do you think?

Reply:
Implemented changes that was supposed by @nva.